subgraph/src/utils/event.ts

```diff
@@ -0,0 +1,9 @@
+import { ethereum } from "@graphprotocol/graph-ts";
+import { log } from "matchstick-as";
```

Comment:
This dependency is not used

This function is already provided by "@amxx/graphprotocol-utils/src/events.ts", so this file is not needed.